website/sidebars.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
             items: [
                 "artist_hosts_hiero",
                 "artist_hosts_nuke",
+                "artist_hosts_nuke_tut",
```

Comment:
We should merge these two into a single page I think. The original nuke docs don't say much, so it should be enough to migrate them nicely to tut page and make it just `Nuke`

Reply:
done